ee/vellum_ee/workflows/display/nodes/vellum/subworkflow_deployment_node.py

```diff
@@ -44,9 +45,14 @@ def serialize(
             for variable_name, variable_value in input_items
         ]
 
-        deployment = display_context.client.workflow_deployments.retrieve(
-            id=str(raise_if_descriptor(node.deployment)),
-        )
+        try:
+            deployment = display_context.client.workflow_deployments.retrieve(
+                id=str(raise_if_descriptor(node.deployment)),
```

Comment:
`str(raise_if_descriptor(node.deployment))` could be consolidated once before the try/except, reused here and below